pkg/controller/admissionchecks/provisioning/controller_test.go

```diff
@@ -264,22 +252,13 @@ func TestReconcile(t *testing.T) {
 		},
 	}
 
-	baseConfig := &kueue.ProvisioningRequestConfig{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "config1",
-		},
-		Spec: kueue.ProvisioningRequestConfigSpec{
-			ProvisioningClassName: "class1",
-			Parameters: map[string]kueue.Parameter{
-				"p1": "v1",
-			},
-			RetryStrategy: &kueue.ProvisioningRequestRetryStrategy{
-				BackoffLimitCount:  ptr.To[int32](3),
-				BackoffBaseSeconds: ptr.To[int32](60),
-				BackoffMaxSeconds:  ptr.To[int32](1800),
-			},
-		},
-	}
+	baseConfigWithoutRetryStrategy := utiltesting.MakeProvisioningRequestConfig("config1").ProvisioningClass("class1").Parameters(map[string]kueue.Parameter{"p1": "v1"})
+
+	baseConfig := baseConfigWithoutRetryStrategy.Clone().RetryStrategy(&kueue.ProvisioningRequestRetryStrategy{
+		BackoffLimitCount:  ptr.To[int32](3),
+		BackoffBaseSeconds: ptr.To[int32](60),
+		BackoffMaxSeconds:  ptr.To[int32](1800),
+	})
```

Comment:
```suggestion
	baseConfig := utiltesting.MakeProvisioningRequestConfig("config1").ProvisioningClass("class1").Parameters(map[string]kueue.Parameter{"p1": "v1"})

	baseConfigWithRetryStrategy := baseConfigWithoutRetryStrategy.Clone().RetryStrategy(&kueue.ProvisioningRequestRetryStrategy{
		BackoffLimitCount:  ptr.To[int32](3),
		BackoffBaseSeconds: ptr.To[int32](60),
		BackoffMaxSeconds:  ptr.To[int32](1800),
	})
```

Sorry, I updated my comment. Maybe like this?